python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -365,6 +368,19 @@ def _check_getter(self):
             raise AttributeError('cannot modify or delete "%s" property (declared as "getter")' % _getName(self))
 
     def __get__(self, obj, cls):
+
+        self_name = _getName(self)
+        GangaObject._acquireChildLock(self_name, obj)
```

Comment:
Hmmm, object is of None type... looks like another bug in the order in which things are constructed vs used